src/TodoList.tsx

```diff
@@ -0,0 +1,106 @@
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+
+interface TodoListProps {
+  todos: Todo[],
+  removeTodo: React.Dispatch<React.SetStateAction<boolean>>,
```

Comment:
The types for removeTodo and editTodo should be (id: number) => void instead of React.Dispatch<React.SetStateAction<boolean>>